### ISSUE TITLE
in chat_ui, speech-to-text mode only activates the mic for about 10 seconds

### ISSUE THREAD

**Issue by AbrahamSanders** [AbrahamSanders/seq2seq-chatbot#6]
There should be a way to configure how long it listens before shutting off.

**Comment by sagarhingal** [comment]
We can keep it to always listen righ? It just then filters out the appropriate questions which the user is asking to him?

**Comment by AbrahamSanders** [comment]
Yeah I think there should be a way to configure it to listen indefinitely. I will need to explore the webkitSpeechRecognition documentation and see if this is possible. If not, there might be a way to capture the event when the speech recognition stops and simply restart it.
